### PR TITLE
feat: add schema

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
feat

**Did you add tests for your changes?**
yes
**If relevant, did you update the documentation?**
No
**Summary**
add Schema

**Does this PR introduce a breaking change?**
Yes, `npm` option is now boolean.

**Other information**
No

### CODE REVIEW

File: src/options.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "title": "install-webpack-plugin options",
```
Comment (alexander-akait) [inline src/options.json:2]: Install Plugin options

File: src/options.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      "$ref": "#/definitions/Prompt"
+    },
+    "quiet": {
+      "$ref": "#/definitions/Quiet"
+    },
+    "yarn": {
+      "$ref": "#/definitions/Yarn"
+    }
+  }
+}
```
Comment (alexander-akait) [inline src/options.json:58]: Other PR (separately) - remove `yarn`/`npm` in favor `packageManager: 'auto' | 'npm' | 'yarn' | 'pnpm'` (auto check lock files like we do in webpack-cli)

Comment (alexander-akait) [inline src/options.json:58]: The next step - remove `peerDependencies` and `dev` options in favor `dependecies: { dev: boolean, peer: boolean }`

Comment (alexander-akait) [inline src/options.json:58]: And finally remove `quiet`, we don't need this, because webpack built-in logger allow to set `level`

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Let's finish 3 task above, then I will do small refactoring and we will do release :star: Great work

**Comment by snitin315** [comment]
Will take care 👍🏻

**Pull request merged**